docs/_writing-tests/testdriver-tutorial.md

```diff
@@ -16,14 +16,14 @@ The first part of this will be browser agnostic, but later we will need to imple
 
 ## Code!
 
-### ```resources/testdriver.js```
+### [resources/testdriver.js](https://github.com/w3c/web-platform-tests/blob/master/resources/testdriver.js)
```

Comment:
I think just "resources/testdriver.js" or possible "/resources/testdriver.js" works here to link to the file in the same repo and revision. Will save some renaming after the repo move.